package-lock.json

```diff
@@ -0,0 +1,9433 @@
+{
```

Comment:
I think generally we don't check in package-lock.json

Reply:
Ok package-lock.json, should be not added to merge request.
It could be use for npm ci  cli command